integration/networking/mac_addr_test.go

```diff
@@ -77,3 +79,154 @@ func TestMACAddrOnRestart(t *testing.T) {
 	assert.Check(t, ctr1MAC != ctr2MAC,
 		"expected containers to have different MAC addresses; got %q for both", ctr1MAC)
 }
+
+// Check that a configured MAC address is restored after a container restart,
+// and after a daemon restart.
+func TestCfgdMACAddrOnRestart(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType == "windows")
```

Comment:
The issue with a MAC address generated by the bridge n/w driver getting reused after a container restart, then becoming a duplicate, doesn't apply on native-Windows - there's no bridge driver. (And these tests are all bridge-network based.)

Do we have any networking integration tests that start containers on Windows? There's probably a gap we should try to fill.

But, I think it has to be out of scope for this bug fix?

Reply:
The tests use the bridge driver, but are the daemon code paths being touched specific to bridge, or Linux? Because it looks suspiciously like the code is cross-platform.
 https://github.com/moby/moby/blob/cff4f20c44a3a7c882ed73934dec6a77246c6323/libnetwork/drivers/windows/windows.go#L539-L545

Reply:
> So, perhaps we can resolve this if I raise an issue saying the regression test is needed, and assign it to myself to have a go at?

I'd be satisfied with editing the "how to verify" section of the PR description to mention manually testing on Windows. Filing an issue to add a Windows regression test would be a huge bonus.

Reply:
Good stuff... that's done.